concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -325,7 +325,7 @@ Castro-2015-608-323	323	to have fun	3239	AMUSE ONESELF (HAVE FUN)	玩	to have fu
 Castro-2015-608-324	324	to ascend	705	GO UP (ASCEND)	上（上坡）	to ascend, to climb, to get on (a bus)
 Castro-2015-608-325	325	to descend	2014	GO DOWN (DESCEND)	下（下坡）	to descend, to come down
 Castro-2015-608-326	326	to climb	1102	CLIMB	爬（爬山）	to climb
-Castro-2015-608-327	327	to cross, pass	1951	PASS	过（经过）	to pass, to go by, to cross
```

Comment:
This could potentially be a problematic case, since the "to pass, to go by" meanings do not refer to the river-crossings we have discussed for `to cross`. I'd leave it as is